clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/view/ChangeCaptureViewWriter.java

```diff
@@ -196,6 +188,7 @@ synchronized private void initializeVeniceWriter(int version) {
     if (veniceWriter != null) {
       return;
     }
+    // We don't want any linger time for this producer
```

Comment:
This looks like a dangling comment... is it from an earlier iteration of the change?

Reply:
good catch, removing